apps/dashboard/app/models/launcher.rb

```diff
@@ -353,6 +355,13 @@ def add_required_fields(form: [], attributes: {})
     add_script_to_form(form: form, attributes: attributes)
   end
 
+  def add_default_fields(form: [], **_args)
+    Configuration.launcher_default_items.each do |default_item|
+      Rails.logger.debug("adding #{default_item} unless #{form.include?(default_item)}")
```

Comment:
Did you want to leave this `debug` in or was it supposed to be yanked?

Reply:
Good catch, I'll remove it.